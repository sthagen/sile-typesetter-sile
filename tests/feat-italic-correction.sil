\begin[papersize=a6]{document}
\use[module=packages.retrograde,target=v0.15.0]
\nofolios
\neverindent
\font[size=15pt]
\use[module=packages.math]
% Italic correction.
% The setting plays at paragraph level
% Disabled:
\set[parameter=typesetter.italicCorrection, value=false]
− (\em{fluff}) ¡\em{fancyful proof}! [\em{puff}] \em{fluff}²

% Enabled:
\set[parameter=typesetter.italicCorrection, value=true]
+ (\em{fluff}) ¡\em{fancyful proof}! [\em{puff}]  \em{fluff}²

% The setting also works in hbox-built content.
% We want to be sure the hbox width is still properly computed
% Hence the | marks.
% Disabled:
\set[parameter=typesetter.italicCorrection, value=false]
− |\hbox{(\em{fluff}) ¡\em{fancyful proof}! [\em{puff}]}|

% Enabled:
\set[parameter=typesetter.italicCorrection, value=true]
+ |\hbox{(\em{fluff}) ¡\em{fancyful proof}! [\em{puff}]}|

% Extra check with French exceptions.
\language[main=fr]{%
% Disabled:
\set[parameter=typesetter.italicCorrection, value=false]
− «\em{fluff}» \em{french proof}! (\em{fluff})

% Enabled:
\set[parameter=typesetter.italicCorrection, value=true]
+ «\em{fluff}» \em{french proof}! (\em{fluff})
}

% We want to insert the correction as a kern or glue depending on context,
% so that line breaking may still occur where it should.
% Disabled
\set[parameter=typesetter.italicCorrection, value=false]
− \em{fluff}! \em{fluff} fluff aaaaaaa aaaa aaaaaa \em{fluff} fluff

% Enabled
\set[parameter=typesetter.italicCorrection, value=true]
+ \em{fluff}! \em{fluff} fluff aaaaaaa aaaa aaaaaa \em{fluff} fluff

% Repeated with another font
\font[family=Cormorant Infant]
\set[parameter=typesetter.italicCorrection, value=false]
− (\em{fluff}) ¡\em{fancyful proof}! [\em{puff}] \em{fluff}²

\set[parameter=typesetter.italicCorrection, value=true]
+ (\em{fluff}) ¡\em{fancyful proof}! [\em{puff}]  \em{fluff}²

\set[parameter=typesetter.italicCorrection, value=false]
− |\hbox{(\em{fluff}) ¡\em{fancyful proof}! [\em{puff}]}|

\set[parameter=typesetter.italicCorrection, value=true]
+ |\hbox{(\em{fluff}) ¡\em{fancyful proof}! [\em{puff}]}|

\language[main=fr]{%
\set[parameter=typesetter.italicCorrection, value=false]
− «\em{fluff}» \em{french proof}! (\em{fluff})

\set[parameter=typesetter.italicCorrection, value=true]
+ «\em{fluff}» \em{french proof}! (\em{fluff})
}

\set[parameter=typesetter.italicCorrection, value=false]
− \em{fluff}! \em{fluff} fluff aaaaaaa aaaa aaaaaa \em{fluff} fluff

\set[parameter=typesetter.italicCorrection, value=true]
+ \em{fluff}! \em{fluff} fluff aaaaaaa aaaa aaaaaa \em{fluff} fluff

\end{document}
