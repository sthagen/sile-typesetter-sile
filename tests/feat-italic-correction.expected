Set paper size 	297.6377985	419.5275636
Begin page
Mx 	14.8819
My 	32.6951
Set font 	Gentium Plus;15;400;;normal;;;LTR
T	2149 w=6.1523	(−)
Mx 	25.0090
T	11 w=4.7534	(()
Mx 	29.7624
Set font 	Gentium Plus;15;400;Italic;normal;;;LTR
T	739 88 747 w=23.2910	(fluff)
Mx 	53.0534
Set font 	Gentium Plus;15;400;;normal;;;LTR
T	12 w=4.7534	())
Mx 	61.7816
T	163 w=4.0796	(¡)
Mx 	65.8612
Set font 	Gentium Plus;15;400;Italic;normal;;;LTR
T	73 68 81 70 92 73 88 79 w=45.9668	(fancyful)
Mx 	115.4414
T	83 85 82 82 73 w=29.1431	(proof)
Mx 	144.5845
Set font 	Gentium Plus;15;400;;normal;;;LTR
T	4 w=4.0796	(!)
Mx 	152.6388
T	62 w=4.6143	([)
Mx 	157.2531
Set font 	Gentium Plus;15;400;Italic;normal;;;LTR
T	83 88 747 w=22.7417	(puff)
Mx 	179.9948
Set font 	Gentium Plus;15;400;;normal;;;LTR
T	64 w=4.6143	(])
Mx 	188.5838
Set font 	Gentium Plus;15;400;Italic;normal;;;LTR
T	739 88 747 w=23.2910	(fluff)
Mx 	211.8748
Set font 	Gentium Plus;15;400;;normal;;;LTR
T	2011 w=4.9292	(²)
Mx 	14.8819
My 	50.6951
T	14 w=6.1523	(+)
Mx 	25.0071
T	11 w=4.7534	(()
Mx 	30.2362
Set font 	Gentium Plus;15;400;Italic;normal;;;LTR
T	739 88 747 w=23.2910	(fluff)
Mx 	56.0468
Set font 	Gentium Plus;15;400;;normal;;;LTR
T	12 w=4.7534	())
Mx 	64.7730
T	163 w=4.0796	(¡)
Mx 	69.5851
Set font 	Gentium Plus;15;400;Italic;normal;;;LTR
T	73 68 81 70 92 73 88 79 w=45.9668	(fancyful)
Mx 	119.1635
T	83 85 82 82 73 w=29.1431	(proof)
Mx 	150.7598
Set font 	Gentium Plus;15;400;;normal;;;LTR
T	4 w=4.0796	(!)
Mx 	158.8123
T	62 w=4.6143	([)
Mx 	163.4560
Set font 	Gentium Plus;15;400;Italic;normal;;;LTR
T	83 88 747 w=22.7417	(puff)
Mx 	188.7172
Set font 	Gentium Plus;15;400;;normal;;;LTR
T	64 w=4.6143	(])
Mx 	197.8102
Set font 	Gentium Plus;15;400;Italic;normal;;;LTR
T	739 88 747 w=23.2910	(fluff)
Mx 	223.3572
Set font 	Gentium Plus;15;400;;normal;;;LTR
T	2011 w=4.9292	(²)
Mx 	14.8819
My 	68.6951
T	2149 w=6.1523	(−)
Mx 	25.0131
T	95 w=3.6621	(|)
Mx 	28.6752
T	11 w=4.7534	(()
Mx 	33.4286
Set font 	Gentium Plus;15;400;Italic;normal;;;LTR
T	739 88 747 w=23.2910	(fluff)
Mx 	56.7196
Set font 	Gentium Plus;15;400;;normal;;;LTR
T	12 w=4.7534	())
Mx 	65.4519
T	163 w=4.0796	(¡)
Mx 	69.5315
Set font 	Gentium Plus;15;400;Italic;normal;;;LTR
T	73 68 81 70 92 73 88 79 w=45.9668	(fancyful)
Mx 	119.1155
T	83 85 82 82 73 w=29.1431	(proof)
Mx 	148.2585
Set font 	Gentium Plus;15;400;;normal;;;LTR
T	4 w=4.0796	(!)
Mx 	156.3170
T	62 w=4.6143	([)
Mx 	160.9312
Set font 	Gentium Plus;15;400;Italic;normal;;;LTR
T	83 88 747 w=22.7417	(puff)
Mx 	183.6729
Set font 	Gentium Plus;15;400;;normal;;;LTR
T	64 w=4.6143	(])
Mx 	188.2872
T	95 w=3.6621	(|)
Mx 	14.8819
My 	86.6951
T	14 w=6.1523	(+)
Mx 	25.0117
T	95 w=3.6621	(|)
Mx 	28.6738
T	11 w=4.7534	(()
Mx 	33.9029
Set font 	Gentium Plus;15;400;Italic;normal;;;LTR
T	739 88 747 w=23.2910	(fluff)
Mx 	59.7134
Set font 	Gentium Plus;15;400;;normal;;;LTR
T	12 w=4.7534	())
Mx 	68.4443
T	163 w=4.0796	(¡)
Mx 	73.2563
Set font 	Gentium Plus;15;400;Italic;normal;;;LTR
T	73 68 81 70 92 73 88 79 w=45.9668	(fancyful)
Mx 	122.8389
T	83 85 82 82 73 w=29.1431	(proof)
Mx 	154.4352
Set font 	Gentium Plus;15;400;;normal;;;LTR
T	4 w=4.0796	(!)
Mx 	162.4922
T	62 w=4.6143	([)
Mx 	167.1359
Set font 	Gentium Plus;15;400;Italic;normal;;;LTR
T	83 88 747 w=22.7417	(puff)
Mx 	192.3972
Set font 	Gentium Plus;15;400;;normal;;;LTR
T	64 w=4.6143	(])
Mx 	197.0114
T	95 w=3.6621	(|)
Mx 	14.8819
My 	104.6951
T	2149 w=6.1523	(−)
Mx 	25.0143
T	169 w=7.2803	(«)
Mx 	35.4705
Set font 	Gentium Plus;15;400;Italic;normal;;;LTR
T	739 88 747 w=23.2910	(fluff)
Mx 	61.9375
Set font 	Gentium Plus;15;400;;normal;;;LTR
T	170 w=7.2803	(»)
Mx 	73.1978
Set font 	Gentium Plus;15;400;Italic;normal;;;LTR
T	73 85 72 81 70 75 w=35.9619	(french)
Mx 	112.7779
T	83 85 82 82 73 w=29.1431	(proof)
Mx 	143.9029
Set font 	Gentium Plus;15;400;;normal;;;LTR
T	4 w=4.0796	(!)
Mx 	151.9626
T	11 w=4.7534	(()
Mx 	156.7160
Set font 	Gentium Plus;15;400;Italic;normal;;;LTR
T	739 88 747 w=23.2910	(fluff)
Mx 	180.0070
Set font 	Gentium Plus;15;400;;normal;;;LTR
T	12 w=4.7534	())
Mx 	14.8819
My 	122.6951
T	14 w=6.1523	(+)
Mx 	25.0138
T	169 w=7.2803	(«)
Mx 	35.4699
Set font 	Gentium Plus;15;400;Italic;normal;;;LTR
T	739 88 747 w=23.2910	(fluff)
Mx 	61.9367
Set font 	Gentium Plus;15;400;;normal;;;LTR
T	170 w=7.2803	(»)
Mx 	73.1709
Set font 	Gentium Plus;15;400;Italic;normal;;;LTR
T	73 85 72 81 70 75 w=35.9619	(french)
Mx 	112.7506
T	83 85 82 82 73 w=29.1431	(proof)
Mx 	143.8756
Set font 	Gentium Plus;15;400;;normal;;;LTR
T	4 w=4.0796	(!)
Mx 	151.9347
T	11 w=4.7534	(()
Mx 	157.1638
Set font 	Gentium Plus;15;400;Italic;normal;;;LTR
T	739 88 747 w=23.2910	(fluff)
Mx 	182.9744
Set font 	Gentium Plus;15;400;;normal;;;LTR
T	12 w=4.7534	())
Mx 	14.8819
My 	140.6951
T	2149 w=6.1523	(−)
Mx 	27.4228
Set font 	Gentium Plus;15;400;Italic;normal;;;LTR
T	739 88 747 w=23.2910	(fluff)
Mx 	50.7138
Set font 	Gentium Plus;15;400;;normal;;;LTR
T	4 w=4.0796	(!)
Mx 	61.1819
Set font 	Gentium Plus;15;400;Italic;normal;;;LTR
T	739 88 747 w=23.2910	(fluff)
Mx 	90.8614
Set font 	Gentium Plus;15;400;;normal;;;LTR
T	739 88 747 w=26.0083	(fluff)
Mx 	123.2583
T	68 68 68 68 68 68 68 w=48.1934	(aaaaaaa)
Mx 	177.8402
T	68 68 68 68 w=27.5391	(aaaa)
Mx 	211.7678
T	68 68 68 68 68 68 w=41.3086	(aaaaaa)
Mx 	259.4649
Set font 	Gentium Plus;15;400;Italic;normal;;;LTR
T	739 88 747 w=23.2910	(fluff)
Mx 	14.8819
My 	158.6951
Set font 	Gentium Plus;15;400;;normal;;;LTR
T	739 88 747 w=26.0083	(fluff)
My 	176.6951
T	14 w=6.1523	(+)
Mx 	26.4319
Set font 	Gentium Plus;15;400;Italic;normal;;;LTR
T	739 88 747 w=23.2910	(fluff)
Mx 	52.1762
Set font 	Gentium Plus;15;400;;normal;;;LTR
T	4 w=4.0796	(!)
Mx 	62.0234
Set font 	Gentium Plus;15;400;Italic;normal;;;LTR
T	739 88 747 w=23.2910	(fluff)
Mx 	93.5412
Set font 	Gentium Plus;15;400;;normal;;;LTR
T	739 88 747 w=26.0083	(fluff)
Mx 	125.2568
T	68 68 68 68 68 68 68 w=48.1934	(aaaaaaa)
Mx 	179.1574
T	68 68 68 68 w=27.5391	(aaaa)
Mx 	212.4037
T	68 68 68 68 68 68 w=41.3086	(aaaaaa)
Mx 	259.4649
Set font 	Gentium Plus;15;400;Italic;normal;;;LTR
T	739 88 747 w=23.2910	(fluff)
Mx 	14.8819
My 	194.6951
Set font 	Gentium Plus;15;400;;normal;;;LTR
T	739 88 747 w=26.0083	(fluff)
My 	212.6951
Set font 	Cormorant Infant;15;400;;normal;;;LTR
T	2531 w=5.9700	(−)
Mx 	25.0773
T	2403 w=4.6350	(()
Mx 	29.7123
Set font 	Cormorant Infant;15;400;Italic;normal;;;LTR
T	718 555 634 816 w=22.2450	(fluff)
Mx 	51.9573
Set font 	Cormorant Infant;15;400;;normal;;;LTR
T	2404 w=4.6350	())
Mx 	60.8177
T	2372 w=3.7050	(¡)
Mx 	64.5227
Set font 	Cormorant Infant;15;400;Italic;normal;;;LTR
Mx 	3.6450
Mx 	6.5100
Mx 	7.0650
Mx 	5.1750
Mx 	6.7500
Mx 	3.9300
Mx 	6.9750
Mx 	3.5550
T	680 a=3.6450 457 a=6.5100 567 a=6.9750 486 a=5.1750 665 a=6.5400 517 a=3.9300 634 a=6.9750 555 a=3.5550	(fancyful)
Mx 	112.3532
Mx 	6.1050
Mx 	4.8300
Mx 	5.9700
Mx 	5.9700
Mx 	3.9300
T	604 a=6.1050 607 a=5.1000 577 a=5.9700 577 a=5.9700 517 a=3.9300	(proof)
Set font 	Cormorant Infant;15;400;;normal;;;LTR
T	2371 w=3.7050	(!)
Mx 	147.0886
T	2407 w=4.1100	([)
Mx 	151.1986
Set font 	Cormorant Infant;15;400;Italic;normal;;;LTR
T	604 634 816 w=20.7150	(puff)
Mx 	171.9136
Set font 	Cormorant Infant;15;400;;normal;;;LTR
T	2408 w=4.1100	(])
Mx 	180.2490
Set font 	Cormorant Infant;15;400;Italic;normal;;;LTR
T	718 555 634 816 w=22.2450	(fluff)
Mx 	202.4940
Set font 	Cormorant Infant;15;400;;normal;;;LTR
T	2328 w=3.7650	(²)
Mx 	14.8819
My 	230.6951
T	2530 w=5.9700	(+)
Mx 	25.0754
T	2403 w=4.6350	(()
Mx 	30.3927
Set font 	Cormorant Infant;15;400;Italic;normal;;;LTR
T	718 555 634 816 w=22.2450	(fluff)
Mx 	54.6027
Set font 	Cormorant Infant;15;400;;normal;;;LTR
T	2404 w=4.6350	())
Mx 	63.4612
T	2372 w=3.7050	(¡)
Mx 	68.7562
Set font 	Cormorant Infant;15;400;Italic;normal;;;LTR
Mx 	3.6450
Mx 	6.5100
Mx 	7.0650
Mx 	5.1750
Mx 	6.7500
Mx 	3.9300
Mx 	6.9750
Mx 	3.5550
T	680 a=3.6450 457 a=6.5100 567 a=6.9750 486 a=5.1750 665 a=6.5400 517 a=3.9300 634 a=6.9750 555 a=3.5550	(fancyful)
Mx 	116.5847
Mx 	6.1050
Mx 	4.8300
Mx 	5.9700
Mx 	5.9700
Mx 	3.9300
T	604 a=6.1050 607 a=5.1000 577 a=5.9700 577 a=5.9700 517 a=3.9300	(proof)
Mx 	145.0921
Set font 	Cormorant Infant;15;400;;normal;;;LTR
T	2371 w=3.7050	(!)
Mx 	153.0206
T	2407 w=4.1100	([)
Mx 	157.5312
Set font 	Cormorant Infant;15;400;Italic;normal;;;LTR
T	604 634 816 w=20.7150	(puff)
Mx 	180.2085
Set font 	Cormorant Infant;15;400;;normal;;;LTR
T	2408 w=4.1100	(])
Mx 	189.2185
Set font 	Cormorant Infant;15;400;Italic;normal;;;LTR
T	718 555 634 816 w=22.2450	(fluff)
Mx 	213.4285
Set font 	Cormorant Infant;15;400;;normal;;;LTR
T	2328 w=3.7650	(²)
Mx 	14.8819
My 	248.6951
T	2531 w=5.9700	(−)
Mx 	25.0817
T	2627 w=2.5200	(|)
Mx 	27.6017
T	2403 w=4.6350	(()
Mx 	32.2367
Set font 	Cormorant Infant;15;400;Italic;normal;;;LTR
T	718 555 634 816 w=22.2450	(fluff)
Mx 	54.4817
Set font 	Cormorant Infant;15;400;;normal;;;LTR
T	2404 w=4.6350	())
Mx 	63.3466
T	2372 w=3.7050	(¡)
Mx 	67.0516
Set font 	Cormorant Infant;15;400;Italic;normal;;;LTR
Mx 	3.6450
Mx 	6.5100
Mx 	7.0650
Mx 	5.1750
Mx 	6.7500
Mx 	3.9300
Mx 	6.9750
Mx 	3.5550
T	680 a=3.6450 457 a=6.5100 567 a=6.9750 486 a=5.1750 665 a=6.5400 517 a=3.9300 634 a=6.9750 555 a=3.5550	(fancyful)
Mx 	114.8864
Mx 	6.1050
Mx 	4.8300
Mx 	5.9700
Mx 	5.9700
Mx 	3.9300
T	604 a=6.1050 607 a=5.1000 577 a=5.9700 577 a=5.9700 517 a=3.9300	(proof)
Set font 	Cormorant Infant;15;400;;normal;;;LTR
T	2371 w=3.7050	(!)
Mx 	149.6263
T	2407 w=4.1100	([)
Mx 	153.7363
Set font 	Cormorant Infant;15;400;Italic;normal;;;LTR
T	604 634 816 w=20.7150	(puff)
Mx 	174.4513
Set font 	Cormorant Infant;15;400;;normal;;;LTR
T	2408 w=4.1100	(])
Mx 	178.5613
T	2627 w=2.5200	(|)
Mx 	14.8819
My 	266.6951
T	2530 w=5.9700	(+)
Mx 	25.0803
T	2627 w=2.5200	(|)
Mx 	27.6003
T	2403 w=4.6350	(()
Mx 	32.9175
Set font 	Cormorant Infant;15;400;Italic;normal;;;LTR
T	718 555 634 816 w=22.2450	(fluff)
Mx 	57.1275
Set font 	Cormorant Infant;15;400;;normal;;;LTR
T	2404 w=4.6350	())
Mx 	65.9909
T	2372 w=3.7050	(¡)
Mx 	71.2859
Set font 	Cormorant Infant;15;400;Italic;normal;;;LTR
Mx 	3.6450
Mx 	6.5100
Mx 	7.0650
Mx 	5.1750
Mx 	6.7500
Mx 	3.9300
Mx 	6.9750
Mx 	3.5550
T	680 a=3.6450 457 a=6.5100 567 a=6.9750 486 a=5.1750 665 a=6.5400 517 a=3.9300 634 a=6.9750 555 a=3.5550	(fancyful)
Mx 	119.1193
Mx 	6.1050
Mx 	4.8300
Mx 	5.9700
Mx 	5.9700
Mx 	3.9300
T	604 a=6.1050 607 a=5.1000 577 a=5.9700 577 a=5.9700 517 a=3.9300	(proof)
Mx 	147.6268
Set font 	Cormorant Infant;15;400;;normal;;;LTR
T	2371 w=3.7050	(!)
Mx 	155.5602
T	2407 w=4.1100	([)
Mx 	160.0708
Set font 	Cormorant Infant;15;400;Italic;normal;;;LTR
T	604 634 816 w=20.7150	(puff)
Mx 	182.7480
Set font 	Cormorant Infant;15;400;;normal;;;LTR
T	2408 w=4.1100	(])
Mx 	186.8580
T	2627 w=2.5200	(|)
Mx 	14.8819
My 	284.6951
T	2531 w=5.9700	(−)
Mx 	25.0824
T	2441 w=6.8400	(«)
Mx 	35.2975
Set font 	Cormorant Infant;15;400;Italic;normal;;;LTR
T	718 555 634 816 w=22.2450	(fluff)
Mx 	60.9177
Set font 	Cormorant Infant;15;400;;normal;;;LTR
T	2442 w=6.8400	(»)
Mx 	71.9882
Set font 	Cormorant Infant;15;400;Italic;normal;;;LTR
Mx 	3.9300
Mx 	4.8300
Mx 	5.1450
Mx 	7.0650
Mx 	5.1750
Mx 	6.6750
T	517 a=3.9300 607 a=5.1000 498 a=5.1450 567 a=6.9750 486 a=5.1750 527 a=6.6750	(french)
Mx 	109.0387
Mx 	6.1050
Mx 	4.8300
Mx 	5.9700
Mx 	5.9700
Mx 	3.9300
T	604 a=6.1050 607 a=5.1000 577 a=5.9700 577 a=5.9700 517 a=3.9300	(proof)
Mx 	137.9497
Set font 	Cormorant Infant;15;400;;normal;;;LTR
T	2371 w=3.7050	(!)
Mx 	145.8851
T	2403 w=4.6350	(()
Mx 	150.5201
Set font 	Cormorant Infant;15;400;Italic;normal;;;LTR
T	718 555 634 816 w=22.2450	(fluff)
Mx 	172.7651
Set font 	Cormorant Infant;15;400;;normal;;;LTR
T	2404 w=4.6350	())
Mx 	14.8819
My 	302.6951
T	2530 w=5.9700	(+)
Mx 	25.0820
T	2441 w=6.8400	(«)
Mx 	35.2970
Set font 	Cormorant Infant;15;400;Italic;normal;;;LTR
T	718 555 634 816 w=22.2450	(fluff)
Mx 	60.9170
Set font 	Cormorant Infant;15;400;;normal;;;LTR
T	2442 w=6.8400	(»)
Mx 	71.7616
Set font 	Cormorant Infant;15;400;Italic;normal;;;LTR
Mx 	3.9300
Mx 	4.8300
Mx 	5.1450
Mx 	7.0650
Mx 	5.1750
Mx 	6.6750
T	517 a=3.9300 607 a=5.1000 498 a=5.1450 567 a=6.9750 486 a=5.1750 527 a=6.6750	(french)
Mx 	108.8116
Mx 	6.1050
Mx 	4.8300
Mx 	5.9700
Mx 	5.9700
Mx 	3.9300
T	604 a=6.1050 607 a=5.1000 577 a=5.9700 577 a=5.9700 517 a=3.9300	(proof)
Mx 	137.7226
Set font 	Cormorant Infant;15;400;;normal;;;LTR
T	2371 w=3.7050	(!)
Mx 	145.6577
T	2403 w=4.6350	(()
Mx 	150.9750
Set font 	Cormorant Infant;15;400;Italic;normal;;;LTR
T	718 555 634 816 w=22.2450	(fluff)
Mx 	175.1850
Set font 	Cormorant Infant;15;400;;normal;;;LTR
T	2404 w=4.6350	())
Mx 	14.8819
My 	320.6951
T	2531 w=5.9700	(−)
Mx 	26.7967
Set font 	Cormorant Infant;15;400;Italic;normal;;;LTR
T	718 555 634 816 w=22.2450	(fluff)
Mx 	49.0417
Set font 	Cormorant Infant;15;400;;normal;;;LTR
T	2371 w=3.7050	(!)
Mx 	58.6916
Set font 	Cormorant Infant;15;400;Italic;normal;;;LTR
T	718 555 634 816 w=22.2450	(fluff)
Mx 	86.8815
Set font 	Cormorant Infant;15;400;;normal;;;LTR
T	952 699 936 w=24.9000	(fluff)
Mx 	117.7263
T	522 522 522 522 522 522 522 w=51.4500	(aaaaaaa)
Mx 	175.1212
T	522 522 522 522 w=29.4000	(aaaa)
Mx 	210.4660
T	522 522 522 522 522 522 w=44.1000	(aaaaaa)
Mx 	260.5109
Set font 	Cormorant Infant;15;400;Italic;normal;;;LTR
T	718 555 634 816 w=22.2450	(fluff)
Mx 	14.8819
My 	338.6951
Set font 	Cormorant Infant;15;400;;normal;;;LTR
T	952 699 936 w=24.9000	(fluff)
My 	356.6951
T	2530 w=5.9700	(+)
Mx 	26.1564
Set font 	Cormorant Infant;15;400;Italic;normal;;;LTR
T	718 555 634 816 w=22.2450	(fluff)
Mx 	50.1038
Set font 	Cormorant Infant;15;400;;normal;;;LTR
T	2371 w=3.7050	(!)
Mx 	59.2925
Set font 	Cormorant Infant;15;400;Italic;normal;;;LTR
T	718 555 634 816 w=22.2450	(fluff)
Mx 	88.9226
Set font 	Cormorant Infant;15;400;;normal;;;LTR
T	952 699 936 w=24.9000	(fluff)
Mx 	119.2428
T	522 522 522 522 522 522 522 w=51.4500	(aaaaaaa)
Mx 	176.1129
T	522 522 522 522 w=29.4000	(aaaa)
Mx 	210.9330
T	522 522 522 522 522 522 w=44.1000	(aaaaaa)
Mx 	260.5109
Set font 	Cormorant Infant;15;400;Italic;normal;;;LTR
T	718 555 634 816 w=22.2450	(fluff)
Mx 	14.8819
My 	374.6951
Set font 	Cormorant Infant;15;400;;normal;;;LTR
T	952 699 936 w=24.9000	(fluff)
End page
Finish
